roles/tas_single_node/templates/configs/etc-hosts.j2

```diff
@@ -1,3 +1,3 @@
-{% for item in ['fulcio','tuf', 'rekor','rekor-search'] %}
+{% for item in ['fulcio','tuf', 'rekor','rekor-search', 'tsa', 'cli-server'] %}
```

Comment:
If we don't end up merging this PR, could you submit a separate PR to add the `tsa` string here? That should be fixed either way.